fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/translator/PhysicalPlanTranslator.java

```diff
@@ -138,8 +147,13 @@ public PlanFragment visitPhysicalOlapScan(
         List<Slot> slotList = olapScan.getOutput();
         PhysicalOlapScan physicalOlapScan = olapScan.getOperator();
         OlapTable olapTable = physicalOlapScan.getTable();
+        List<Expr> execConjunctsList = physicalOlapScan
+                .getExpressions()
+                .stream()
+                .map(e -> ExpressionTranslator.convert(e, context)).collect(Collectors.toList());
         TupleDescriptor tupleDescriptor = generateTupleDesc(slotList, context, olapTable);
         OlapScanNode olapScanNode = new OlapScanNode(context.nextNodeId(), tupleDescriptor, olapTable.getName());
```

Comment:
All to thrift properties should be assigned in the translator, otherwise be cannot execute the query.
Please check the properties of each PlanNode and add.

Reply:
done